.github/actions/install-debian-packages/action.yml

```diff
@@ -16,18 +12,25 @@ inputs:
 runs:
   using: "composite"
   steps:
+    - name: Cache debian packages
+      uses: actions/cache@v2
+      id: cache-debs
+      with:
+        path: "~/cache-debs"
+        key: v3
```

Comment:
Perhaps we can add a comment here indicating this should be updated when the packages are changed. Just to avoid somebody else hitting the same problem you hit with the cache.

Reply:
I wrote it here:
https://github.com/kinvolk/inspektor-gadget/pull/529/files/ced6dc7b5df83b9b0c66a1053bb4e62278d2be34#diff-6aa1982dd6b6db00df35c70d94604a381bf22ea49c53a97aeb03bcd05e1a74dbR27
But I can put it over the key field if needed.

Reply:
I missed it. It seems a good place to have it there.